packages/common/src/types/explore.ts

```diff
@@ -90,3 +90,8 @@ export type Table = TableBase & {
     lineageGraph: LineageGraph; // DAG structure representing the lineage of the table
     source?: Source;
 };
+
+export enum CustomViewType {
+    VIRTUAL = 'virtual',
+    PERSISTENT = 'persistent',
```

Comment:
what do you think?

Reply:
or perhaps `write_back`

Reply:
done! https://github.com/lightdash/lightdash/pull/11728/commits/8d7ff6a055271e6a7c4de0272998a22f976e5f48